README.md

```diff
@@ -17,9 +17,9 @@ Installation
 
 We recommend the [Anaconda](https://www.anaconda.com/download/) Python
 distribution. We require that you use Python 3.
-Besides `numpy`, `scipy` and `matplotlib` (which are included in the standard
-Anaconda installation), you will need to install the most recent version of MNE
-using the `pip` tool:
+Besides `numpy` and `scipy` (which are included in the standard Anaconda
```

Comment:
shouldn't we also update the CI if we do this? otherwise we won't know ...

Reply:
CIs are already updated. When I did it, I just missed to remove Matplotlib from the README